custom/enikshay/integrations/nikshay/tests/test_repeaters.py

```diff
@@ -88,6 +90,9 @@ def _create_nikshay_registered_case(self):
         )
         self.create_case(nikshay_registered_case)
 
+    def update_case_with(self, case_id, case_attrs):
```

Comment:
I still think you should remove the word `with`, or change it to be `update_case_with_attributes` although that seems redundant.